package/scie-pants.toml

```diff
@@ -9,7 +9,7 @@ repo = "https://github.com/pantsbuild/scie-pants"
 [lift.ptex]
 id = "ptex"
 version = "0.7.0"
-lazy_argv1 = "{scie.env.PANTS_BOOTSTRAP_URLS={scie.lift}}"
+argv1 = "{scie.env.PANTS_BOOTSTRAP_URLS={scie.lift}}"
```

Comment:
This one was a real bug. You're supposed to be able to point at a private repo with the env var and this would have made that not work.

Reply:
There was additional confusion here re `SCIE=inspect`, but this may have been a bug sighting: https://pantsbuild.slack.com/archives/C046T6T9U/p1692201968356429?thread_ts=1692132192.942339&cid=C046T6T9U

Reply:
And, in fact, see the update here: https://github.com/pantsbuild/scie-pants/issues/243

Reply:
Ah, good catch. I've confirmed that this indeed fixes the behaviour there.